src/main/java/net/minecraftforge/common/loot/LootModifierManager.java

```diff
@@ -154,4 +161,17 @@ private IGlobalLootModifier deserializeModifier(ResourceLocation location, JsonE
         return registeredLootModifiers.values();
     }
 
+    @SubscribeEvent
+    public static void onResourceReload(AddReloadListenerEvent event)
```

Comment:
Forge has its own internal handler, so this shouldn't have this weird meta instance thing...

Reply:
used here https://github.com/MinecraftForge/MinecraftForge/blob/1.16.x/src/main/java/net/minecraftforge/common/ForgeHooks.java#L1169 but commented for now. I can put this in the InternalHandler if its better suited.

Reply:
Yes this should be moved to the internal handler as there is no sense in exposing a modder facing API if we can avoid it when it comes to internal instances of things.